.github/workflows/release.yml

```diff
@@ -8,15 +8,14 @@ jobs:
     runs-on: ubuntu-latest
     if: github.repository == 'blockprotocol/blockprotocol'
     steps:
-      - uses: actions/checkout@v2
+      - uses: actions/checkout@v3
 
-      - uses: actions/setup-node@v2
+      - uses: actions/setup-node@v3
         with:
           node-version: 16
           cache: yarn
 
-      - name: Install deps and build (with cache)
-        uses: bahmutov/npm-install@v1
+      - run: yarn install
```

Comment:
I’ve copied this yaml from a `changesets` example, but have realised that we don’t need [bahmutov/npm-install](https://github.com/bahmutov/npm-install) here. `actions/setup-node` should take care of the Node cache.

Unfortunately, it’s impossible to check this change immediately, but if this pipeline fails on `main`, it won’t cause any damage.